### ISSUE TITLE
SCSS is not properly recompiled 

### ISSUE THREAD

**Issue by adi8i** [gantry/gantry5#1332]
```
// Grid Row and Column Setup
.g-block {
	@include breakpoint(mobile-only) {
		@include flex(0 100%);
	}
}

body [class*="size-"] {
	@include breakpoint(mobile-only) {
		@include flex(0 100%);
	}
}
```

Is compiled as `flex: 0;` which cause issues on mobile.

![scss](https://cloud.githubusercontent.com/assets/9744973/15799139/7bcffb6e-2a5b-11e6-92fb-34e099c65184.PNG)

![scss 2](https://cloud.githubusercontent.com/assets/9744973/15799161/edf84c50-2a5b-11e6-89fc-c107b1c8b3c2.PNG)

**Comment by w00fz** [comment]
Is this the CI Build?

// Djamil Legato // From iPhone //

**Comment by adi8i** [comment]
Yes, sorry I forgot to mention.

**Comment by w00fz** [comment]
Does it help if you recompile in Base?

// Djamil Legato // From iPhone //

**Comment by adi8i** [comment]
No, weird, the code in nucleus looks fine.

**Comment by w00fz** [comment]
It could well be an issue, we upgraded both the scss compiler and bourbon to latest after a year, we might still have a few issues that needs investigating.

// Djamil Legato // From iPhone //

**Comment by w00fz** [comment]
This one kinda stinks. Looks like the issue is caused by the current SCSS compiler in `master`.
`@include flex(X Y);` excludes Y and only considers X. However bourbon mixin is totally legit to take into account 2 values.

Either way I worked around it by passing the 2 individual CSS rules, rather than concatenating them.

Please @adi8i make sure you open issues like this if you spot other CSS issues, 5.3 is quite a huge upgrade with SCSS compiler, Bourbon and Inheritance.

Thanks!

**Comment by mahagr** [comment]
Is this issue in the SCSS compiler? We need to make sure there's a bug report for this.

**Comment by hexplor** [comment]
This is going to be a HELL with our themes. 
Topaz: 30 around flex mixins with various particles
Interstellar: 36
etc etc.. all templates will have to be reworked because of this

**Comment by newkind** [comment]
We *MUSTN'T* release 5.3 without this getting fixed as *ALL* of our G5 products are using this mixin and there's no way we're going to release updates/fix everything before releasing 5.3. People are also not very good in updating their installations so this will break tons of sites.

**Comment by mahagr** [comment]
Alright, after having a deeper look into this issue, there's a workaround we can use to prevent the bug from happening.
